presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2593,6 +2593,11 @@ public void revokeTablePrivileges(ConnectorSession session, SchemaTableName sche
         return toCompletableFuture(stagingFileCommitter.commitFiles(session, handle.getSchemaName(), handle.getTableName(), getPartitionUpdates(fragments)));
     }
 
+    public FunctionMetadataManager getFunctionMetadataManager()
```

Comment:
this should be injected into HiveConnector. No need to open an interface.

Reply:
@sachdevs +1. Is there any particular reason to introduce this method?